linera-client/src/client_options.rs

```diff
@@ -964,6 +964,14 @@ pub enum DatabaseToolCommand {
         #[arg(long = "storage")]
         storage_config: String,
     },
+
+    /// List the root keys of the database
+    #[command(name = "list_root_keys")]
+    ListRootKeys {
```

Comment:
In `linera-client`, we know that the "root keys" are (basically) chain ids. So shouldn't we list the chain ids instead?

Reply:
And similarly we should list ApplicationId, BlobId, etc as such.

Reply:
Btw you may need to add conversion function in `linera-storage` for this.

Reply:
Yes, this is why I wanted to do it in a subsequent PR.